src/algorithms/discretevoronoi.jl

```diff
@@ -0,0 +1,32 @@
+"""
+    DiscreteVoronoi
+
+Assigns a value to each patch using a 1-NN algorithmm with `n` initial clusters
+and `1` neighbors. The default is to use three clusters.
+"""
+struct DiscreteVoronoi <: NeutralLandscapeMaker
+    n::Int64
+    function DiscreteVoronoi(n::Int64)
+        @assert n > 0
+        new(n)
+    end
+end
+
+DiscreteVoronoi() = DiscreteVoronoi(3)
```

Comment:
why not just add 3 as a default value in the internal constructor in line 9?

Reply:
oh nice. I see